yarn.lock

```diff
@@ -3077,6 +3077,7 @@ __metadata:
     react: ^16.13.1
     react-dom: ^16.13.1
     react-dropzone: ^11.1.0
+    react-error-boundary: ^3.0.2
```

Comment:
Would rather prefer not pulling in a dep here. It is not that complex since React has it as a base - https://github.com/polkadot-js/apps/blob/master/packages/react-components/src/ErrorBoundary.tsx

Reply:
Sounds good, I thought it's simple, but maintained by high profile devs so that should have covered us pretty well, but having our own class is also fine, I didn't know we had it in apps.